website/cue/reference/releases/0.33.0.cue

```diff
@@ -0,0 +1,691 @@
+package metadata
+
+releases: "0.33.0": {
+	date:     "2023-09-26"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		The Vector team is pleased to announce version 0.33.0!
+
+		Be sure to check out the [upgrade guide](/highlights/2023-09-26-0-33-0-upgrade-guide) for
+		breaking changes in this release.
+
+		This release marks the switch of the default configuration language for Vector from TOML to
+		YAML. We continue to support both (as well as JSON), but the documentation will prefer YAML
+		in its configuration examples. We think this will lead to more legible configurations as
+		well as more consistency with Vector's Helm chart which requires YAML configuration. See the
+		associated [release highlight](/highlights/2023-08-30-yaml-default-format) for more details
+		on the motivation for this change.
+
+		In addition to the usual enhancements and bug fixes, this release includes also includes:
+
+		- A new AWS SNS sink
+
+		"""
+
+	known_issues: []
+
+	changelog: [
+		{
+			type: "feat"
+			scopes: ["exec source"]
+			description: """
+				The `exec` source now allows customization of the environment variables exposed to
+				the subprocess via two new options:
+
+				- `clear_environment` to remove environment variables propagated from Vector process
+				- `environment` to set custom environment variables for the subprocess
+				"""
+			contributors: ["hhromic"]
+			pr_numbers: [18223]
+		},
+		{
+			type: "fix"
+			scopes: ["socket source"]
+			description: """
+				The type definition for the `port` field included on events emitted from the
+				`socket` source is now correctly marked as an integer rather than a string. This
+				removes unnecessary casting when the `log_namespacing` feature is enabled.
+				"""
+			pr_numbers: [18180]
+		},
+		{
+			type: "fix"
+			scopes: ["elasticsearch sink"]
+			description: """
+				The `elasticsearch` sink now correctly ignores the `pipeline` configuration when it
+				is an empty string (`""`) to allow disabling this option. Previously it would result
+				in failed Elasticsearch requests due to an empty pipeline name being passed.
+				"""
+			pr_numbers: [18248]
+		},
+		{
+			type: "enhancement"
+			scopes: ["cli"]
+			description: """
+				Vector now has the ability to disable OpenSSL probing for remote certificates via
+				`--openssl-no-probe` (or `VECTOR_OPENSSL_NO_PROBE`).
+				"""
+			contributors: ["hhromic"]
+			pr_numbers: [18229]
+		},
+		{
+			type: "fix"
+			scopes: ["kubernetes_logs source"]
+			description: """
+				Fix log event generation from the `kubernetes_logs` source when `log_namespacing` is
+				enabled. Previously it would emit empty log events.
+				"""
+			contributors: ["hhromic"]
+			pr_numbers: [18229]
```

Comment:
This wasn't me 🙈 :)
The PR number is also duplicated from the above entry.

Reply:
Doh! Copy/paste error. Thanks for flagging.